src/Epub.ts

```diff
@@ -266,6 +278,20 @@ export default class Epub {
     return ncxStr ? Epub.parseXmlString<NCX>(ncxStr, NCX) : undefined;
   }
 
+  /**
+   * Convert the relative href relative to the opfPath, that's the OEBPS or ops folder
+   * This is to keep things consistant and make opfPath the main root folder for encrypted contents
+   */
+  static resolvePathToOpfPath(
```

Comment:
I would just change fetcher.resolveRelativePath to accept undefined. However, it needs to have a type overload so that we know if we pass in a string, we get back a string but if we pass in a `string | undefined` we get back a `string | undefined`. Something like this:

```ts
abstract resolveRelativePath(from: string, to: string): string;
abstract resolveRelativePath(from: string, to: string | undefined): string | undefined;
```